### DISCUSSION TITLE
How?

### DISCUSSION THREAD

**Discussion by Nathan208619** [ohio-software-development/.github#16] [Q&A] [answered]
Code?

**Comment by BP-2** [comment, marked as answer]
Simply `def sieve_of_eratosthenes(n):
    primes = [True] * (n+1)
    primes[0] = primes[1] = False

    for i in range(2, int(n**0.5)+1):
        if primes[i]:
            for j in range(i*i, n+1, i):
                primes[j] = False

    return [i for i in range(n+1) if primes[i]]

# Example usage:
lower_limit = 10
upper_limit = 50
print(f"Prime numbers between {lower_limit} and {upper_limit}:")
print(sieve_of_eratosthenes(upper_limit))
`
